pygmt/base_plotting.py

```diff
@@ -261,12 +263,12 @@ def grdcontour(self, grid, **kwargs):
             - Optional label modifiers can be specified as a single string
               ``'[annot_int]+e'``  or with a list of options
               ``([annot_int], 'e', 'f10p', 'gred')``.
-        L : str or list of 2 ints
+        limit : str or list of 2 ints
             Do no draw contours below `low` or above `high`, specify as string
             ``'[low]/[high]'``  or list ``[low,high]``.
-        Q : string or int
+        cut : string or int
             Do not draw contours with less than `cut` number of points.
-        S : string or int
+        resample : string or int
             Resample smoothing factor.
         {J}
         {R}
```

Comment:
Need to add `{U}` for timestamp to `grdcontour` docstring here.

Reply:
Ideally, we should add V to all the modules, and B, U, X, Y, and p to all plotting modules. I personally prefer to leave that to another separate PR.

Reply:
Maybe add the `{U}` for now since it's aliased already.

Agree to add `B, U, X, Y` later. Was thinking if there was a way to add it to all of them via the `_preprocess` function.

Reply:
Done.